tools/readme/readme.md.tmpl

```diff
@@ -39,6 +39,8 @@ Run the `help` command and see available commands:
 elastic-package help
 ```
 
+Some sub-commands are Docker-based, check you also have Docker installed. In case you are using Podman, check [this guide](./docs/howto/use_podman.md) to make it compatible.
```

Comment:
```suggestion
Some sub-commands are Docker-based, check you also have Docker installed. In case you are using Podman Desktop, check [this guide](./docs/howto/use_podman.md) to make it compatible.
```